tasks/__init__.py

```diff
@@ -86,11 +86,13 @@ def git_logs(ctx, branch=None):
 
 
 @task
-def apiserver(ctx, port=8000, wait=True, host='127.0.0.1'):
+def apiserver(ctx, port=8000, wait=True, autoreload=True, host='127.0.0.1'):
```

Comment:
Looks like the docs refer to this as "[Automatic boolean inverse flags](http://docs.pyinvoke.org/en/0.13.0/concepts/cli/intro.html#automatic-boolean-inverse-flags)."